webpack.config.js

```diff
@@ -101,7 +102,9 @@ module.exports = [{
   },
   postcss: function() {
     return [
-      require('autoprefixer')
+      require('autoprefixer'),
+      require('postcss-custom-properties'),
```

Comment:
Yes, it replaces the custom properties with hard-coded values. Then the color function processes that down as well into standard output. As we can start shipping custom properties we just remove this processor and ship them down.